src/cpp/src/scheduler.hpp

```diff
@@ -321,7 +321,7 @@ class Scheduler {
                 if (!m_config.enable_prefix_caching)
                     OPENVINO_ASSERT(sequence_group->get_context_len() == 0);
 
-                size_t num_available_tokens_in_megabatch = m_config.max_num_batched_tokens - scheduler_output.m_total_num_scheduled_tokens;
+                int64_t num_available_tokens_in_megabatch = m_config.max_num_batched_tokens - scheduler_output.m_total_num_scheduled_tokens;
```

Comment:
looks like the logic with max_sequence_len is obsolete remnants of padding computation.

Let's drop it and keep size_t here.